mb_1200px_caa.user.js

```diff
@@ -21,22 +21,22 @@ var ca_page = document.querySelector('div#content');
 var ca_items = ca_page.querySelectorAll('div.artwork-cont');
 
 ca_items.forEach(function(ca_item) {
-  /* Use 1200px “thumbnails” for the pop‐ups/previews */
-  var popup_link = ca_item.querySelector('a.artwork-image');
-  popup_link.href = popup_link.href.replace(/\.[a-z]+$/, '-1200.jpg');
+    /* Use 1200px “thumbnails” for the pop‐ups/previews */
+    let popup_link = ca_item.querySelector('a.artwork-image');
```

Comment:
"Back in my day" `let` was not a thing. I’ll read up on `let` and be sure to not use `var` in the future. :p